### DISCUSSION TITLE
Welcome to data_utils Discussions!

### DISCUSSION THREAD

**Discussion by sdhakal07** [sdhakal07/data_crow#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sigdelsanjog** [comment]
@sdhakal07 
Shall we develop an ETL in python as a separate feature line at Datacrow?

Or shall we start a new project itself? 

I have started a scrapping tool here:

1. [Scrapemore-backend](https://github.com/sigdelsanjog/scrapemore-backend)
2. [scrapemore-frontend](https://github.com/sigdelsanjog/scrapemore)
